cms/static/cms/js/require-config.js

```diff
@@ -50,8 +50,8 @@
             'jquery.immediateDescendents': 'coffee/src/jquery.immediateDescendents',
             'datepair': 'js/vendor/timepicker/datepair',
             'date': 'js/vendor/date',
-            'moment': 'js/vendor/moment.min',
-            'moment-with-locales': 'js/vendor/moment-with-locales.min',
+            'moment': 'common/js/vendor/moment-with-locales.min',
+            'moment-timezone': 'common/js/vendor/moment-timezone-with-data.min',
```

Comment:
Can you provide a high-level explanation on this PR for why these `moment` library changes were needed?